frontend-public/src/HOC/AuthenticationGuard.js

```diff
@@ -15,7 +15,9 @@ export const AuthenticationGuard = (isPublic) => (WrappedComponent) => {
     componentDidMount() {
       const token = localStorage.getItem("jwt");
       if (token && !this.props.isAuthenticated) {
-        this.props.getUserInfoFromToken(token);
+        this.props.getUserInfoFromToken(token).catch(() => {
+          // Silently fail
```

Comment:
I like to create a const doNothing = () => {}; as it makes itself describing without comments.

Reply:
Fair, but seems a waste if its only used once. Maybe we should add it to constants.js?

Reply:
Lodash provides a `noop` function 🤷‍♂️

Also, we use `() => {}` all the time as a default in `defaultProps`, so we'll definitely be able to use this constant or lodash function many times